### PR TITLE
Summit push to prod beta

### PR DESCRIPTION
**Pull request by PanSpagetka**

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -506,21 +825,29 @@ settings:
...
   frontend:
     sub_apps:
-      - id: rbac
+      - id: my-user-access
         default: true
+      - id: rbac
       - id: sources
+      - id: integrations
+      - id: notifications
+      - id: connector
```
Comment (ryelo) [inline main.yml]: @PanSpagetka can we remove this line for now?

**Review by ryelo — CHANGES_REQUESTED** [review]

**Comment by ryelo** [comment]
@kruai any objections to bringing in all of these files?

**Comment by kruai** [comment]
It looks like the source branch was ci-beta, and you're trying to merge to prod-beta... I'd recommend closing this one and starting from prod-beta

**Comment by PanSpagetka** [comment]
Closing this in favor of #508

**Pull request closed without merging**